src/tools/PopulateDB.java

```diff
@@ -28,6 +32,81 @@ private static void makeUserTable(Connection conn) {
         }
     }
 
+    private static void makeDriverTable(Connection conn) {
+        String createDriverTable = "CREATE TABLE drivers (" +
+                "id INT PRIMARY KEY," +
+                "name TEXT," +
+                "carNumber INT," +
+                "sex CHAR(1)," +
+                "location TEXT," +
+                "rating REAL," +
+                "reviews TEXT," +
+                "FOREIGN KEY(carNumber) REFERENCES cars(licenseNumber)," +                
+                ");";
+        Statement stmt = null;
+        try {
+            stmt = conn.createStatement();
+            stmt.execute(createDriverTable);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void makeCarTable(Connection conn) {
+        String createCarTable = "CREATE TABLE cars (" +
+                "licenseNumber TEXT PRIMARY KEY," +
+                "model TEXT," +
+                "costPerKm TEXT," +
+                "CarType TEXT," +
+                "speed INT" +
+                ");";
+        Statement stmt = null;
+        try {
+            stmt = conn.createStatement();
+            stmt.execute(createCarTable);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void makeLocationTable(Connection conn) {
+        String createLocationTable = "CREATE TABLE locations (" +
+                "name TEXT PRIMARY KEY," +
+                "x INT," +
+                "y INT," +
+                ");";
+        Statement stmt = null;
+        try {
+            stmt = conn.createStatement();
+            stmt.execute(createLocationTable);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void makeRideTable(Connection conn) {
+        String createRideTable = "CREATE TABLE rides (" +
+                "id INT PRIMARY KEY," +
+                "userEmail TEXT ," +
+                "driver TEXT," +
+                "source TEXT," +
+                "destination TEXT," +
+                "car TEXT," +
+                "startTime TEXT," +
```

Comment:
Isn't there a datetime field in our database?

Reply:
SQLite doesnt support datetime.It's better to cut the bullshit and save as text only.